code/LINDA/LINDA_turf_tile.dm

```diff
@@ -270,14 +270,10 @@ turf/simulated/proc/share_temperature_mutual_solid(turf/simulated/sharer, conduc
 
 
 atom/movable/var/pressure_resistance = 5
-atom/movable/var/last_forced_movement = 0
 
 atom/movable/proc/experience_pressure_difference(pressure_difference, direction)
-	if(last_forced_movement >= air_master.current_cycle)
```

Comment:
I'm pretty sure this was in place specifically to prevent things from flying across the room multiple times per tick